### DISCUSSION TITLE
Public Roadmap

### DISCUSSION THREAD

**Discussion by sdenike** [mastodon/mastodon#22789] [General]
Based on some previous searches, I know this has been brought up, but I am sure I am not alone in wishing there was a public roadmap of what features were coming down the pipe, as well as perhaps a release schedule (even if rough). I know that WordPress has a public timeline even where they have a list of proposed release dates for each release that they try to stick to; of course, there are times when dates change but this could give all of us operators an idea of when to expect/plan upgrades to platforms to support possible new features, etc.  Just a thought.

**Comment by mjankowski** [comment]
After years of being unmaintained and out of date, this was recently updated - https://joinmastodon.org/roadmap - and hypothetically will continue to be updated. Enjoy.
